kafka-client/src/main/java/dev/responsive/kafka/clients/EndOffsetsPoller.java

```diff
@@ -88,10 +89,10 @@ public EndOffsetsPoller(
   }
 
   public synchronized Listener addForThread(final String threadId) {
-    LOG.info("add end offset metrics for thread {}", threadId);
+    LOG.debug("Adding end offset metrics for thread {}", threadId);
```

Comment:
For some context, the Metrics implementation from Kafka that we use has the "adding/removing metric X" logging all the way down to TRACE. Debug seems like a reasonable compromise imo